src/main/java/com/epam/javacourses2016/task16/Cell.java

```diff
@@ -0,0 +1,45 @@
+package com.epam.javacourses2016.task16;
+
+import com.epam.javacourses2016.Point2D;
+
+/**
+ * Created by Рамиль on 30.11.2016.
+ */
+public class Cell {
+
+    private final Point2D CENTER;
+    private final Point2D TOP_LEFT_CORNER;
+    private final Point2D BOT_LEFT_CORNER;
+    private final Point2D TOP_RIGHT_CORNER;
+    private final Point2D BOT_RIGHT_CORNER;
```

Comment:
But these guys are not constants, they are just final fields and should not be written in CAPITAL.